src/app/components/Core/Songs/SongList/SongList.scss

```diff
@@ -108,10 +115,36 @@
       }
     }
 
-    .songDuration {
+    .songRightSide {
       width: 60px;
       align-items: center;
       color: $tertiary_color;
+
+      .songActions {
+        display: none;
+        color: #fe2851;
```

Comment:
Do . we not have a var for that ?

Reply:
We do, my bad. Shall add this in before merging (and after I push my next PR).